docs/query_attribute.rst

```diff
@@ -330,3 +337,20 @@ The following example returns all the lithological descriptions of boreholes tha
 
     lithologische_beschrijvingen = ls.search(query=And([Join(boringen, 'pkey_boring'),
                                                         PropertyIsEqualTo('betrouwbaarheid_interpretatie', 'goed')]))
+
+The following example gets borehole information based on a search for groundwater filters:
+
+::
+
+    from pydov.util.query import Join
+
+    from pydov.search.boring import BoringSearch
+    from pydov.search.grondwaterfilter import GrondwaterFilterSearch
+
+    fs = GrondwaterFilterSearch()
+    bs = BoringSearch()
+
+    filters = fs.search(location=WithinDistance(Point(96540, 186900), 10, 'km'),
+                        return_fields=('pkey_filter', 'boringfiche'))
```

Comment:
Could we illustrate the usage by providing an example of the top rows of the `filters` dataframe?

Reply:
Good idea, I added the top rows from both dataframes.